python_ta/debug/snapshot.py

```diff
@@ -44,3 +46,50 @@ def snapshot():
         frame = frame.f_back
 
     return variables
+
+
+def snapshot_to_json(snapshot_data: List[Dict]):
```

Comment:
Add a return type here. 

But also, return a `dict`, not a `str`. You don't need to use `json.dumps` here, it's a bit more useful to just have the `dict` and then let the user decide what to do with it.

Reply:
Hello @david-yz-liu, thank you for your feedback. I'm currently considering the rationale behind returning a dictionary in our implementation. Initially, I've been accumulating the objects/frames to be drawn in a list. Transitioning to a dictionary prompts the question: what would be the appropriate key for each corresponding value?

Reply:
Oh yes, you can return a `list`, that's good. I meant more that you shouldn't return a string.